app/presenters/two_factor_login_options_presenter.rb

```diff
@@ -42,15 +42,23 @@ def options
     configurations.group_by(&:class).flat_map { |klass, set| klass.selection_presenters(set) }
   end
 
-  def should_display_account_reset_or_cancel_link?
+  def ial2?
```

Comment:
Could we rename this to capture the policy we are actually testing? Perhaps `should_allow_recovery_by_reverification?`

Reply:
I kept your old name